mne/viz/_brain/_brain.py

```diff
@@ -934,6 +967,75 @@ def _configure_point_picking(self):
             self._on_pick
         )
 
+    def _configure_trace_mode(self):
+        from ...label import _read_annot_cands
```

Comment:
I think there is a bug in _read_annot_cands

see:

```
ipdb> cc = 'lh.random30.annot'
ipdb> cc.lstrip('lh.').lstrip('rh.').rstrip('.annot')
'andom30'
```

it should return `random30`

Reply:
What do you think about:

https://github.com/mne-tools/mne-python/blob/1ab0863cdfae6e4951b1027b897fc470d3537849/mne/label.py#L1883-L1884

Reply:
works great !